sdk/notificationhubs/notification-hubs/package.json

```diff
@@ -119,24 +119,25 @@
     "karma-edge-launcher": "^0.4.2",
     "karma-env-preprocessor": "^0.1.1",
     "karma-firefox-launcher": "^2.1.2",
+    "karma-ie-launcher": "^1.0.0",
```

Comment:
we don't support IE now. so this should not be needed?

Reply:
Removed